packages/react-native/src/posthog-rn.ts

```diff
@@ -179,6 +201,11 @@ export class PostHog extends PostHogCore {
 
       this.setupBootstrap(options)
 
+      // Set default person properties for flags if enabled
+      if (this._setDefaultPersonProperties) {
+        this._setDefaultPersonPropertiesForFlags()
```

Comment:
similar to https://github.com/PostHog/posthog-js/pull/2564/files#r2518097170 since flags will be reloaded after this method call